client-course-schedulizer/src/components/reuseables/AddSectionPopover/AddSectionPopover.tsx

```diff
@@ -61,35 +73,71 @@ export const AddSectionPopover = () => {
           <span>{/* TODO: add error messages? */}</span>
         </Grid>
       </Grid>
       <Grid container spacing={SPACING}>
         {/* TODO: Dropdown for instructors with option to add new one */}
-        <GridItemTextField label="Instructor" register={register} />
-        <GridItemTextField label="Student Hours" register={register} />
-        <GridItemTextField label="Faculty Hours" register={register} />
+        <GridItemTextField
+          label="Instructor"
+          register={register}
+          value={values?.section.instructors.join()}
+        />
         {/* TODO: Dropdown for rooms with option to add new one */}
-        <GridItemTextField label="Location" register={register} />
-        <GridItemTextField label="Room Capacity" register={register} />
+        <GridItemTextField label="Location" register={register} value={locationValue} />
+        <GridItemTextField
+          label="Room Capacity"
+          register={register}
+          textFieldProps={{ name: "roomCapacity" }}
+          value={(values?.meeting.location.roomCapacity || "").toString()}
+        />
+        <GridItemTextField
+          label="Faculty Hours"
+          register={register}
+          textFieldProps={{ name: "facultyHours" }}
+          value={(values?.section.facultyHours || values?.course.facultyHours || "").toString()}
+        />
+        <GridItemTextField
+          label="Student Hours"
+          register={register}
+          textFieldProps={{ name: "studentHours" }}
+          value={(values?.section.studentHours || values?.course.studentHours || "").toString()}
+        />
       </Grid>
       <Grid container spacing={SPACING}>
-        <GridItemTextField label="Anticipated Size" register={register} />
-        <GridItemTextField label="Global Max" register={register} />
-        <GridItemTextField label="Local Max" register={register} />
+        <GridItemTextField
+          label="Anticipated Size"
+          register={register}
+          textFieldProps={{ name: "anticipatedSize" }}
+          value={(values?.section.anticipatedSize || "").toString()}
+        />
+        <GridItemTextField
+          label="Local Max"
+          register={register}
+          textFieldProps={{ name: "localMax" }}
+          value={(values?.section.localMax || "").toString()}
+        />
+        <GridItemTextField
+          label="Global Max"
+          register={register}
+          textFieldProps={{ name: "globalMax" }}
+          value={(values?.section.globalMax || "").toString()}
+        />
         <GridItemTextField
           label="Start Time"
           register={register}
-          textFieldProps={{
-            defaultValue: "08:00",
-            fullWidth: true,
-            type: "time",
-          }}
+          textFieldProps={{ fullWidth: true, name: "startTime", type: "time" }}
+          value={
+            values?.meeting.startTime
+              ? moment(values?.meeting.startTime, "h:mma").format("HH:mm")
```

Comment:
This will help to make things more consistent with the code in #58 
```suggestion
              ? moment(values?.meeting.startTime, "h:mm A").format("HH:mm")
```